packages/config/src/handlers/forms/validation.ts

```diff
@@ -433,6 +449,20 @@ const form = z.object({
           }
         )
     )
+    .refine(
+      (sections) =>
+        sections
+          .find(({ id }) => id === 'review')
+          ?.groups.some(({ id }) => id === 'review-view-group'),
+      `A "review" section is required in form configuration. It can optionally include SIGNATURE fields. An example configuration can be found in our Farajaland reference implementation.`
+    )
+    .refine(
+      (sections) =>
+        sections
+          .find(({ id }) => id === 'preview')
+          ?.groups.some(({ id }) => id === 'preview-view-group'),
+      `A "preview" section is required in form configuration. It can optionally include SIGNATURE fields. An example configuration can be found in our Farajaland reference implementation.`
+    )
```

Comment:
Nice nice nice! To my previous comment about the validation flowing to the components, could we rather just assume the forms are correct as they are actually validated here?